content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -339,12 +344,19 @@ Tremolo Security's [OpenUnison](https://openunison.github.io/).
 
 For an identity provider to work with Kubernetes it must:
 
-1.  Support [OpenID connect discovery](https://openid.net/specs/openid-connect-discovery-1_0.html); not all do.
-2.  Run in TLS with non-obsolete ciphers
-3.  Have a CA signed certificate (even if the CA is not a commercial CA or is self signed)
+1. Support [OpenID connect discovery](https://openid.net/specs/openid-connect-discovery-1_0.html); not all do.
+2. Run in TLS with non-obsolete ciphers
+3. Have a CA signed certificate (even if the CA is not a commercial CA or is self signed)
 
-A note about requirement #3 above, requiring a CA signed certificate.  If you deploy your own identity provider (as opposed to one of the cloud providers like Google or Microsoft) you MUST have your identity provider's web server certificate signed by a certificate with the `CA` flag set to `TRUE`, even if it is self signed.  This is due to GoLang's TLS client implementation being very strict to the standards around certificate validation.  If you don't have a CA handy, you can use [this script](https://github.com/dexidp/dex/blob/master/examples/k8s/gencert.sh) from the Dex team to create a simple CA and a signed certificate and key pair.
-Or you can use [this similar script](https://raw.githubusercontent.com/TremoloSecurity/openunison-qs-kubernetes/master/src/main/bash/makessl.sh) that generates SHA256 certs with a longer life and larger key size.
+A note about requirement #3 above, requiring a CA signed certificate.  If you deploy your own
+identity provider (as opposed to one of the cloud providers like Google or Microsoft) you MUST
+have your identity provider's web server certificate signed by a certificate with the `CA` flag
+set to `TRUE`, even if it is self signed.  This is due to GoLang's TLS client implementation
+being very strict to the standards around certificate validation.  If you don't have a CA handy,
+you can use [this script](https://github.com/dexidp/dex/blob/master/examples/k8s/gencert.sh) from
```

Comment:
```suggestion
you can use [this script](https://github.com/dexidp/dex/blob/master/examples/k8s/gencert.sh) from
```
According to styleguide, https://kubernetes.io/docs/contribute/style/style-guide/#links see/here/this is not recommended.

Reply:
how about this:

```
you can use the [gencert script](https://github.com/dexidp/dex/blob/master/examples/k8s/gencert.sh) from
```

Reply:
looks promising.